src/components/factlist.scss

```diff
@@ -0,0 +1,49 @@
+@use "variables" as *;
+
+.facts-wrapper {
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+}
+
+.facts-header {
+  text-align: center;
+  padding: 0;
+}
+
+.section-header {
+  font-size: 8rem;
+  text-align: center;
+}
+
+.fact-header {
+  font-size: 5rem;
+  text-align: left;
+}
+
+.facts-list {
+  display: grid;
+  row-gap: 1rem;
+  grid-template-columns: auto;
+}
+
+li {
+  list-style: none;
+}
```

Comment:
This will apply to all `li` elements on the whole site, which you don't want - replace with a specific class name applied to the element

Reply:
or, nest this inside the above, so you get
```scss
.facts-list {
  ...
  li {
    ...
  }
}
```

Reply:
factlist.scss redundant file from wip. File not needed or used so deleted.